cas-server-support-oauth/src/main/java/org/jasig/cas/config/OAuthConfiguration.java

```diff
@@ -51,14 +73,35 @@
     public Config oauthSecConfig() {
         final CasClient oauthCasClient = new CasClient(this.casLoginUrl);
         oauthCasClient.setName(CAS_OAUTH_CLIENT);
+        oauthCasClient.setCallbackUrlResolver((url, context) -> {
```

Comment:
OK. So there is a bug, but I think the solution is to define the callback authorize service to return all attributes: https://github.com/Jasig/cas/blob/master/cas-server-support-oauth/src/main/java/org/apereo/cas/OAuthApplicationContextWrapper.java#L48, isn't it?